### ISSUE TITLE
I am unable of merging from unstable branch into master branch

### ISSUE THREAD

**Issue by gusbemacbe** [gusbemacbe/suru-plus#49]
Hello @SmartFinn!

Is it safe to force to merge the unstable branch into the master branch when there are conflicts? GitHub said it can not merge automatically, but I can create a pull request.

Is it possible to remove the master branch and to create new branch from unstable branch, renaming to "master"?

**Comment by SmartFinn** [comment]
Try to merge it with the command:

```
git merge --strategy=ours unstable
```

**Comment by SmartFinn** [comment]
Sorry, I was much hurry. The command is wrong. Reset the master branch if you executed the command with `git reset --hard origin/master`

**Comment by gusbemacbe** [comment]
Do you mean:

```
git checkout master
git reset --hard origin/master
```

is it?

**Comment by gusbemacbe** [comment]
What if ...

```
git clone -b unstable --single-branch https://github.com/gusbemacbe/suru-plus
git checkout master
git reset --hard origin/master
``` 
?

**Comment by SmartFinn** [comment]
I see you merged the unstable branch. How you did it?

**Comment by gusbemacbe** [comment]
I found the tricks and I risked, but fortunately there were no problems. Here:

1. I clicked "5 branches"
2. I clicked "Change default branch" and changed "master" to "beta" as default.
3. I saw that the branch "master" could be deleted without restriction and harms and I deleted the branch.
4. At Code, I open "Branch: beta" and created the new branch from "beta", naming "master" without restrictions and problems.
5. I changed from beta to master as default.
